.github/workflows/ci.yml

```diff
@@ -88,23 +102,21 @@ jobs:
           echo '@trustbloc:registry=https://npm.pkg.github.com' >> ~/.npmrc
           make bdd-test-wallet-server
         env:
-          DOCKER_PASSWORD: $(DOCKER_PASSWORD)
-        displayName: Run HTTP Server (wallet-server)
+          DOCKER_PASSWORD: ${{ secrets.CR_PAT }}
 
-  - job: Publish
-    dependsOn:
-      - Checks
-      - UnitTest
-      - BDDTestJS
-      - BDDTestHTTPServer
-    condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
-    pool:
-      vmImage: ubuntu-20.04
-    timeoutInMinutes: 30
+  Publish:
```

Comment:
Curious if we are going to do the GitHub releases too?

Reply:
Yes

Reply:
https://github.com/trustbloc/wallet/issues/844